reference/docs-conceptual/learn/application-control.md

```diff
@@ -0,0 +1,51 @@
+---
+description: You can secure your PowerShell environments on Windows using Windows Defender Application Control (WDAC).
+ms.date: 05/23/2023
+title: Using Windows Defender Application Control
+---
+# Using Windows Defender Application Control
+
+Windows 10 includes two technologies, [Windows Defender Application Control (WDAC)][04] and
+[AppLocker][01] that can be used for application control. They allow you to create a lockdown
+experience to meet your organization's specific scenarios and requirements.
+
+> [!NOTE]
+> When it comes to [choosing between WDAC or AppLocker][03] it's generally recommended that
+> customers implement application control using WDAC rather than AppLocker. WDAC is undergoing
+> continual improvements and will be getting added support from Microsoft management platforms.
+> Although AppLocker will continue to receive security fixes, it will not receive feature
+> enhancements.
```

Comment:
```suggestion
> When [choosing between WDAC or AppLocker][03], it's generally recommended that customers
> implement application control using WDAC rather than AppLocker. Microsoft is continually
> improving WDAC and Microsoft management platforms are extending their support for WDAC. Although
> AppLocker will continue to receive security fixes, it won't receive feature enhancements.
```